lib/logstash/outputs/scalyr.rb

```diff
@@ -104,6 +105,16 @@ class LogStash::Outputs::Scalyr < LogStash::Outputs::Base
   # An int containing the compression level of compression to use, from 1-9. Defaults to 6
   config :compression_level, :validate => :number, :default => 6
 
+  # How often to log and report status metrics to Scalyr. Defaults to every 5
+  # minutes.
+  config :status_report_interval, :validate => :number, :default => 300
+
+  # Whether or not to count status event uploads in the statistics such as request latency etc.
+  config :record_stats_for_status, :validate => :boolean, :default => false
+
+  # Parser to attach to status events
+  config :status_parser, :validate => :string, :default => "logstash_parser"
```

Comment:
:+1:

Reply:
Also, probably better to name this parser logstash_metrics or similar

Reply:
And to make it easier to parse / write parser for it, we could remove commas between key value pairs. That would be  a breaking change though (in case anyone wrote parser for previous format in the past)